codecademy/practice_make_perfect.py

```diff
@@ -0,0 +1,105 @@
+import sys
+import math
+
+if sys.version_info > (3,):
+    long = int
+    unicode = str
+
+
+def is_int(x):
+    # function compute an integer number or not, and return false or true
+    assert isinstance(x, (int, float, long)) and type(x) is not bool, "{} passed the wrong data type".format(x)
+    if x - round(x) > 0:
+        return False
+    else:
+        if x < 0 and x - round(x) < 0:
+            return False
+        else:
+            return True
+# END is_int
+
+
+def factorial(x):
+    assert isinstance(x, (int, float, long)) and type(x) is not bool, "{} passed the wrong data type".format(x)
+    x = abs(x)
+    if x == 1 or x == 0:
+        print(1)
+        return 1
+    else:
+        i = 1
+        for n in range(1, x + 1):
+            i = n * i
+            print(i)
+    return i
+# END factorial
+
+
+def reverse(text):
+    assert isinstance(text, (str, unicode)), "{} passed the wrong data type".format(text)
+    a = ""
+    j = 0
+    for i in range(len(text)):
+        j = j - 1
+        a += text[len(text) + j]
+    return a
+# END reverse
+
+
+score = {"a": 1, "c": 3, "b": 3, "e": 1, "d": 2, "g": 2,
```

Comment:
since score is used only inside `scrabble_score()` i'd put it inside that function.